src/main/java/rizzler/Deadline.java

```diff
@@ -0,0 +1,68 @@
+package rizzler;
```

Comment:
Maybe it's better to put the Task to its own package? For example, rizzler.task package